### ISSUE TITLE
Units of currents if voltage clamped is very high

### ISSUE THREAD

**Issue by deezer257** [jaxleyverse/jaxley#491]
First, I used an older jaxley version since I can't record any currents in the newest jaxley version. I cloned the pull request #473, so it was the version: 0.4.0.

If I voltage clamp and record the resulting current, the current should be in mA according to this website since I think a voltage clamp is not defined as a point process: https://jaxleyverse.github.io/jaxley/faq/question_01/. If I convert the current from mA to pA by multiplying it with e9, I notice that the current is always off by e6. So, I think the current is in nA in the first place, which is used in a point process.

``` python
from jaxley.channels.hh import HH
import jaxley as jx
import jax.numpy as jnp
import matplotlib.pyplot as plt

# Create a new dummy network
comp = jx.Compartment()
branch = jx.Branch(comp, nseg=1)
cell = jx.Cell([branch], [-1])

cell.insert(HH())

v = jnp.zeros(int(1000 / 0.1)) + 30

cell.delete_recordings()
cell.delete_clamps('v')

# Apply voltage clamp
cell.clamp('v', v, False)

to_records = ['v', HH().current_name]

# Record variables
for rec in to_records:
    cell.record(rec, verbose=False)

# Integrate the cell model
s = jx.integrate(cell, delta_t=dt, t_max=time_max) 

# Convert mA to pA
current_pA = s[1,:] * 1e9
 
fig, ax = plt.subplots(2, 1, figsize=(10, 6))
ax[0].plot(s[0,:])
ax[1].plot(current_pA)

# Deactivate scientific notation
ax[0].ticklabel_format(style='plain', axis='y')
ax[1].ticklabel_format(style='plain', axis='y')
´´´

**Comment by michaeldeistler** [comment]
Membrane currents are **not** point processes.

The unit of the current had indeed been wrong though. This is fixed already, but not yet released on PyPI: https://github.com/jaxleyverse/jaxley/pull/461

Before the fix, the unit was uA/cm2. After the fix, it is mA/cm2.

**Comment by michaeldeistler** [comment]
I will close this for now, but feel free to re-open if anything remains unclear. Thanks for reporting it!
